src/components/ViewComments.jsx

```diff
@@ -1,15 +1,22 @@
 import { getComments } from "../api";
 import { useEffect, useState } from "react";
-import { useParams } from "react-router-dom";
+import { Link, useParams } from "react-router-dom";
 import { postComment } from "../api";
+import { deleteComment } from "../api";
+
 
 const ViewComments = () => {
 
     const [comments, setComments] = useState();
     const {article_id} = useParams();
     const [isLoading, setIsLoading] = useState(true);
+    const [isDeleting, setIsDeleting] = useState(false)
     const [newComment, setNewComment] = useState([]);
-    const [err, setErr] = useState(null);
+    const [posterr, setPostErr] = useState(null);
```

Comment:
Optional -

I like this approach, you could save a few lines of code and an extra if statement if you combined the deleteerr and posterr states so that the error message is added to the state by whichever functions are currently toggling these.